### PR TITLE
Make Verbosity as injectable

### PR DESCRIPTION
**Pull request by kastahov**
| Q             | A
| ------------- | ---
| Bugfix?       | ❌
| Breaks BC?    | ❌ 
| New feature?  | ✔️

### CODE REVIEW

File: src/Exceptions/src/Verbosity.php [hunk 1]
```diff
@@ -4,9 +4,23 @@
...
+#[ProvideFrom(method: 'detect')]
+enum Verbosity: int implements InjectableEnumInterface
 {
     case BASIC = 0;
     case VERBOSE = 1;
     case DEBUG = 2;
+
+    public static function detect(EnvironmentInterface $environment): self
+    {
+        return match ($environment->get('VERBOSITY')) {
```
Comment (butschster) [inline src/Exceptions/src/Verbosity.php:20]:
```suggestion
        return match (\strtolower((string) $environment->get('VERBOSITY_LEVEL'))) {
```

File: src/Exceptions/tests/VerbosityTest.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        $env->shouldReceive('get')->with('VERBOSITY')->andReturn($name);
+
+        $enum = Verbosity::detect($env);
+
+        $this->assertSame($expected, $enum);
+    }
+
+    public function envVariablesDataProvider(): array
+    {
+        return [
```
Comment (butschster) [inline src/Exceptions/tests/VerbosityTest.php:26]: Add test with `null`
Add test with camelCase words to check strtolower function

File: src/Http/src/Middleware/ErrorHandlerMiddleware/SuppressErrorsInterface.php [hunk 3]
```diff
@@ -4,6 +4,9 @@
 
 namespace Spiral\Http\Middleware\ErrorHandlerMiddleware;
 
+/**
+ * @deprecated
```
Comment (butschster) [inline src/Http/src/Middleware/ErrorHandlerMiddleware/SuppressErrorsInterface.php:8]: Remove this deprecation. We won't deprecate it

**Review by butschster — CHANGES_REQUESTED** [review]

File: src/Exceptions/src/Verbosity.php [hunk 4]
```diff
@@ -4,9 +4,23 @@
...
+            'basic' => self::BASIC,
+            'debug' => self::DEBUG,
+            default => self::VERBOSE
```
Comment (roxblnfk) [inline src/Exceptions/src/Verbosity.php:23]: We should support ENUM values here (0, 1, 2)

**Review by butschster — APPROVED** [review]

File: src/Http/src/Middleware/ErrorHandlerMiddleware.php [hunk 5]
```diff
@@ -30,6 +31,7 @@ public function __construct(
         private readonly RendererInterface $renderer,
         private readonly ResponseFactoryInterface $responseFactory,
         private readonly ExceptionHandlerInterface $errorHandler,
+        private readonly Verbosity $verbosity
```
Comment (roxblnfk) [inline src/Http/src/Middleware/ErrorHandlerMiddleware.php:34]: The parameter should have a default value

File: src/Exceptions/tests/VerbosityTest.php [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+            ['basic', Verbosity::BASIC],
+            [0, Verbosity::BASIC],
+            ['Basic', Verbosity::BASIC],
+            ['debug', Verbosity::DEBUG],
+            [2, Verbosity::DEBUG],
+            ['invalid', Verbosity::VERBOSE],
+            ['', Verbosity::VERBOSE],
+            [null, Verbosity::VERBOSE],
+            [true, Verbosity::VERBOSE],
+            [false, Verbosity::VERBOSE],
```
Comment (roxblnfk) [inline src/Exceptions/tests/VerbosityTest.php:39]: Please add cases with the values `VERBOSE` and `1`
